VideoLocker/src/test/java/org/edx/mobile/test/UtilTests.java

```diff
@@ -32,4 +34,10 @@ public void testHostAndUrls() throws Exception {
         assertFalse(BrowserUtil.isUrlOfHost("https://fake-domain.com/edx.org/", host));
         assertFalse(BrowserUtil.isUrlOfHost("https://fake-domain.com/xyz/", host));
     }
+
+    @Test
+    public void testSha1Encryption() {
```

Comment:
I think it would be better to have separate test suites for each utility class for better organization (both for the reader and the test code itself). In which case the `convertToHex()` method should be tested as well I guess.